HousingSearchListener/serverless.yml

```diff
@@ -0,0 +1,118 @@
+service: housing-search-listener
+provider:
+  name: aws
+  runtime: dotnetcore3.1
+  memorySize: 2048
+  tracing:
+    lambda: true
+    apiGateway: true
+  vpc: ${self:custom.vpc.${opt:stage}}
+  stage: ${opt:stage}
+  region: eu-west-2
+
+package:
+  artifact: ./bin/release/netcoreapp3.1/housing-search-listener.zip
+
+functions:
+  PersonApi:
+    name: ${self:service}-${self:provider.stage}
+    handler: HousingSearchListener::HousingSearchListener.LambdaEntryPoint::FunctionHandlerAsync
+    role: lambdaExecutionRole
+    environment:      
+      ASPNETCORE_ENVIRONMENT:  ${ssm:/housing-tl/${self:provider.stage}/aspnetcore-environment}
+    events:
+      - http:
+          path: /{proxy+}
+          method: ANY
+          authorizer:
+            arn: ${ssm:/api-authenticator/${self:provider.stage}/arn}
+            type: request
+            resultTtlInSeconds: 0
+            identitySource: method.request.header.Authorization
+            managedExternally: true       
+          cors: true
```

Comment:
This will need to change to hook up the SQS queue with the function. But I guess that's a later PR.